modules/special-rules/pages/hit-and-run.adoc

```diff
@@ -8,13 +8,13 @@ Some forces have honed tactics in which they make a lightning assault, seemingly
 
 Where your detachment has the {hit-and-run} ability:
 
-* After a close combat or firefight with this detachment, you can always make a retreat move -- even if you win.
+* After a close combat or firefight with this detachment, you can always make a Retreat move -- even if you win.
 +
-+[TODO: As written, it seems that they can make the retreat move even if they were charged by the enemy first. Is this intended? It doesn't seem to fit with the idea of 'hit and run'.]+
++{blank}[TODO: As written, it seems that they can make the Retreat move even if they were charged by the enemy first. Is this intended? It doesn't seem to fit with the idea of 'hit and run', it's more like 'hit, or get hit, and run'...!]+
```

Comment:
Note, I'm really just asking the question here, I don't feel all that strongly about it — I can see that the 'who actually charged on this occasion' bit is somewhat arbitrary given the nature of the game mechanics and turn structure.

Reply:
My reply must have got lost in an old pull request somewhere!

The original text just specifies 'after a close combar or firefight, even if the win', so there's no requirement for the detachment to be the one assaulting. I don't see any background issues as it still represents the detachment fading away from an assault.

Reply:
Thematically, I do think that there is a difference between being the ones to initiate an assault and then fading away, and being assaulted and then fading away. Mechanically, we can draw a parallel in which side actually makes the assault/charge move — which would make sense as the 'Hit' part of _Hit and Run_.

So, I wonder if the ability should trigger only when the controlling player made the assault/charge move into firefight/close combat.

The problem is that the [original text](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/SpecialistsAndSpecialAbilities/HitAndRun.htm) is somewhat ambiguous, because the main paragraph uses thematic phrasing and casts White Scars as the prime movers throughout — and it doesn't actually describe things in terms of game mechanics and player actions:

> The White Scars attack from nowhere, unleash a devastating volley of fire or brief charge into close combat, and then disappear as suddenly as they arrived. To represent this, White Scars detachments can always make a retreat move after a close combat or firefight, even if they win. This enables them to attack the enemy and then withdraw to safety to avoid an unfavourable assault or firefight from other enemy detachments close by.

To my eyes, you can easily interpret thus:

- "attack from nowhere" — they initiate the assault/the _Hit and Run_ controlling player makes the assault move
- "brief charge" — they initiate the close combat/the _Hit and Run_ controlling player makes the charge move
- "they arrived" — they made the assault/the _Hit and Run_ controlling player made the assault move
- "enables them to attack the enemy and then withdraw" — they initiate the assault and then get to withdraw/the _Hit and Run_ controlling players makes the assault move and may make a Retreat move even if they win (which we might take to imply that the latter is dependent upon the former)

If you say that this interpretation is too restrictive or otherwise problematic in the game mechanics or game balance, and that _Hit and Run_ controlling players should be able to make a Retreat move after they win whether nor not they made the assault move to initiate that assault, that's fine with me — I'm just asking the question because it seems ambiguous enough to allow argument for either interpretation.

At the extreme, think of this possibility if the _Hit and Run_ controlling player can have it both ways:

1. The _Hit and Run_ detachment is in a static, defensible position. (Not what you'd normally expect, but possible.)
2. The opponent assaults this detachment, moving their forces into firefight/close combat.
3. The _Hit and Run_ player wins the assault, repelling the enemy forces — and then uses the ability to leave that static position.

This seems odd to me. We can explain it away if we have to, but given the ambiguity of the original text and the thematic oddity of this situation, I figure it bears some consideration.

Reply:
> The problem is that the [original text](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/SpecialistsAndSpecialAbilities/HitAndRun.htm) is somewhat ambiguous, because the main paragraph uses thematic phrasing and casts White Scars as the prime movers throughout — and it doesn't actually describe things in terms of game mechanics and player actions:

Ah! I was going by the bit that **is** written in terms of game mechanics, and ignoring everything that (to me) looks like background or description (mechanics marked in bold):

>> _The White Scars attack from nowhere, unleash a devastating volley of fire or brief charge into close combat, and then disappear as suddenly as they arrived. To represent this,_ **White Scars detachments can always make a retreat move after a close combat or firefight, even if they win**. _This enables them to attack the enemy and then withdraw to safety to avoid an unfavourable assault or firefight from other enemy detachments close by._

This gives us:

(Hit and Run) detachments can always make a retreat move after a close combat or firefight, even if they win.

> If you say that this interpretation is too restrictive or otherwise problematic in the game mechanics or game balance, and that _Hit and Run_ controlling players should be able to make a Retreat move after they win whether nor not they made the assault move to initiate that assault, that's fine with me — I'm just asking the question because it seems ambiguous enough to allow argument for either interpretation.

For me, the bit that's actually written in terms of game mechanics isn't ambiguous.

It's simple, it works, I don't think we add anything to the game by making the ability more complex or more restricted.

> At the extreme, think of this possibility if the _Hit and Run_ controlling player can have it both ways:
> 
> 1. The _Hit and Run_ detachment is in a static, defensible position. (Not what you'd normally expect, but possible.)
> 2. The opponent assaults this detachment, moving their forces into firefight/close combat.
> 3. The _Hit and Run_ player wins the assault, repelling the enemy forces — and then uses the ability to leave that static position.
> 
> This seems odd to me. We can explain it away if we have to, but given the ambiguity of the original text and the thematic oddity of this situation, I figure it bears some consideration.

It's possible, as you say, but it's very unlikely as the ability is only on detachments that are fully mechanised and/or made up of very fast units.

But even then, that doesn't sound like a thematic issue. White Scars temporarily taking up a defensive position to lure the attacker into a bad assault, and then feinting away again after the assault sounds completely in character.

Reply:
> Ah! I was going by the bit that is written in terms of game mechanics, and ignoring everything that (to me) looks like background or description (mechanics marked in bold):
> [...]
> For me, the bit that's actually written in terms of game mechanics isn't ambiguous.

My point is that it wouldn't be the first time that GW text failed to mention an important detail in the actual explicit rules bit that we had to infer from the surrounding descriptive bit — and that they actually **intended** that interpretation. Thus, ambiguous.

I get that we can't spend all our time hypothesising about things that GW could've failed to mention — but I raised the question here specifically because because all the surrounding descriptive text is very much in the sense that the _Hit and Run_ detachment initiated the assault.

> It's simple, it works, I don't think we add anything to the game by making the ability more complex or more restricted.

I don't think that restriction would complicate matters any more than many other simple restrictions that exist elsewhere in the game. **But**, if you don't think this is an occasion where GW made a mistake of the type I mention above, and you don't think there's any benefit to such a restriction anyway, then that's fine with me. In that case we can delete this TODO note.

```suggestion
```